components/Panel/VotePanel/Discussion.tsx

```diff
@@ -132,13 +152,17 @@ const handleWordBreak = css`
 `;
 
 const Text = styled.p`
-  ${handleWordBreak}
   font: var(--text-md);
   &:not(:last-child) {
     margin-bottom: 15px;
   }
 `;
 
+const MessageText = styled(Text)`
+  word-wrap: break-word;
+  word-break: break-word;
```

Comment:
break word instead of break all on paragraph tags in the markdown, also to fix  word wrap